.scripts/checkEverything.ts

```diff
@@ -4,6 +4,6 @@ import { checkConstantsVersion } from "./checkConstantsVersion";
 checkEverything({
   additionalChecks: {
     name: "Constants.ts Version",
-    check: checkConstantsVersion
-  }
+    check: checkConstantsVersion,
```

Comment:
in prettier v2.0, trailingComma:  "es5" is now default. It makes adding new entries for multi-line lists easier.  https://prettier.io/docs/en/options.html#trailing-commas